splink/linker.py

```diff
@@ -1600,6 +1603,15 @@ def compute_number_of_comparisons_generated_by_blocking_rule(
         res = self._execute_sql_pipeline().as_record_dict()[0]
         return res["count_of_pairwise_comparisons_generated"]
 
+    def count_pairwise_comparisons_generated_by_prediction_blocking_rules(
+        self, df_predict
+    ):
+        sql = mtch_key_anlysis(df_predict)
```

Comment:
Needs a docstring, here's a template:

Reply:
```
        """_summary_

        Args:
            df_predict (_type_): _description_

        Examples:
            >>> 

        Returns:
            _type_: _description_
        """   
```

Reply:
See [here](https://github.com/moj-analytical-services/splink/blob/a56a4850567b0791145ff8e47e0fa691ae18a449/splink/linker.py#L1572) for example